bleachbit/Cleaner.py

```diff
@@ -497,7 +501,8 @@ def __init__(self):
             # the uninstallers for software updates.
             self.add_option('updates', _('Update uninstallers'), _(
                 'Delete uninstallers for Microsoft updates including hotfixes, service packs, and Internet Explorer updates'))
-
+        elif 'posix' == os.name and sys.platform.startswith('darwin'):
```

Comment:
The attached picture does not make it. I did a search for `recycle_bin` and there is no Mac-specific code (besides `add_option`).